test/javascript/components/common/Accordion.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { Accordion } from '../../../../app/javascript/components/common/Accordion'
+
+test('collapsed accordion', () => {
+  render(
+    <Accordion index="accordion" open={false}>
```

Comment:
The tests in this file refer to an accordion being collapsed or expanded, but the property of the accordion is named `open`. It might be good to also use the collapsed/expanded terminology for the property of the accordion.

Reply:
I will change the test descriptions to use `open`.